front-end/src/components/NewLoan/NewLoan.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react'
+import { StyleSheet, Text, Platform } from 'react-native'
+import { Container, Header, Left, Body, Title, Content, Button, Icon, Form, Item, Input, Label, Card, Picker } from 'native-base'
+
+import PaymentDetails from './PaymentDetails'
+import ProductDescription from './ProductDescription'
+
+export default function NewLoan() {
+    const [selected, setSelected] = useState();
```

Comment:
@amelieykw this is not going to pass the linter, we should use 2 spaces for indentation